cltk/text_reuse/levenshtein.py

```diff
@@ -22,6 +12,85 @@ def __init__(self):
         """Initialize class. Currently empty."""
         return
 
+    @staticmethod
+    def Levenshtein_Distance(w1, w2):
```

Comment:
Perhaps we should consider using this library:
https://pypi.org/project/python-Levenshtein/
either way, there needs to be unit tests for this.

Reply:
Sorry, I just noticed the requested changes( will probably write the unit tests once I get back home). The main idea was minimizing dependencies and as WA tends to be fairly simple and rarely (if at all) stray away from the standard algorithms, I decided that writing my own code wouldn't hurt and it would open up the code to further modifications. Anyway, I could scrap it all together and use the library if it is preferred.

Reply:
So I've thought this over: you can absolutely add an implementation of a function such as Levenstein, however you must provide a thorough number of tests as people will actually read your function implementation and compare it against others.

Usually, I do like the idea of offloading a library function to a well-supported, active 3rd party, such as the: https://github.com/ztane/python-Levenshtein library, but, *but* that library mostly offers speed improvements (which although nice, isn't CLTK's primary concern). And sadly, although that source code is excellent, it does not readily provide a plethora of examples that people can immediately learn from. 

So document your implementation and provide lots of tests.